### DISCUSSION TITLE
Script Check: Antivirus - Verify Status, Showing Antivirus check not supported on this OS. Returning Exit Code 2

### DISCUSSION THREAD

**Discussion by motechsolutions** [amidaware/tacticalrmm#1414] [General]
Hi All

Im using the "Script Check Antivirus" check on one of my sites and some PC's are showing this in the "Last Output" "Antivirus check not supported on this OS. Returning Exit Code 2"
The PC's are running Win10 64bit and have Eset business Endpoint Antivirus.

The strange thing is this is only happening to some PC's at this one site, all my other sites with Eset AV are fine.

Note: the agent installed is the Tactical RMM digital signed version

Has anyone got any idea's on this?

Regards
Mick

**Comment by silversword411** [comment]
That script only queries built in Windows Defender. If you've replaced it with ESET, that script won't work. Contact ESET and see if they can share a powershell script that will monitor their AV for alerts and post it here. We'll see about getting it in the Community Scripts library for others
